api/AuthenticationAssertion.json

```diff
@@ -46,6 +46,9 @@
           "safari_ios": {
             "version_added": false
           },
+          "samsunginternet_android": {
+            "version_added": "9.0"
```

Comment:
This was enabled behind a flag in Chrome Android. Was this enabled by default in Samsung Internet?